cmd/integration_test/build/buildspecs/cloudstack-test-eks-a-cli.yml

```diff
@@ -54,6 +54,9 @@ env:
     T_CLOUDSTACK_TEMPLATE_REDHAT_1_22: "cloudstack_ci_beta_connection:template_redhat_1_22"
     T_CLOUDSTACK_TEMPLATE_REDHAT_1_23: "cloudstack_ci_beta_connection:template_redhat_1_23"
     T_CLOUDSTACK_TEMPLATE_REDHAT_1_24: "cloudstack_ci_beta_connection:template_redhat_1_24"
+    T_CLOUDSTACK_TEMPLATE_REDHAT_1_25: "cloudstack_ci_beta_connection:template_redhat_1_25"
+    T_CLOUDSTACK_TEMPLATE_REDHAT_1_26: "cloudstack_ci_beta_connection:template_redhat_1_26"
+    T_CLOUDSTACK_TEMPLATE_REDHAT_1_27: "cloudstack_ci_beta_connection:template_redhat_1_27"
```

Comment:
Let's not add this. The tests have logic to determine the template name based on K8S and EKS-A release/bundle version. We should rely on that. 
We also want to remove the other CloudStack templates from here eventually.